tesseract_core/sdk/templates/Dockerfile.base

```diff
@@ -70,6 +68,9 @@ ENV TESSERACT_NAME="{{ config.name }}" \
 COPY --from=build_stage /python-env /python-env
 COPY "{{ tesseract_source_directory }}/tesseract_api.py" ${TESSERACT_API_PATH}
```

Comment:
fyi, you can use `COPY --chown=tesseractor:tesseractor` here instead of running `chown` separately on the file.

Reply:
@dionhaefner thanks, that's a good tip. do you prefer doing it file-by-file instead of `chown -R` on the whole directory at the end? I'm happy with either approach.

Reply:
Yes, I believe this is a best practice that has slightly more well-behaved Docker caching and whatnot.